drbert/data/dataset_readers.py

```diff
@@ -94,3 +104,16 @@ class SequenceLabellingDatasetReader(DatasetReader):
 
 class NLIDatasetReader(DatasetReader):
     pass
+
+if __name__ == "__main__":
+   
+    # Params for Cohort Task
```

Comment:
This looks like it was used locally for testing? We should remove it from the repo